torch/optim/lr_scheduler.py

```diff
@@ -59,9 +78,26 @@ def load_state_dict(self, state_dict):
         """
         self.__dict__.update(state_dict)
 
-    def get_lr(self):
+    def get_computed_values(self):
+        # Return last computed learning rate by current scheduler
+        return self._last_computed_values
```

Comment:
So for example, a way `get_computed_values` differs from `get_lr` is that it's not valid to call `get_computed_values` on a LR scheduler which has not been stepped once. Maybe it would be safer and more obvious to our users if we raised an error in this case, instead of returning a list of NaNs (which may not be detected until much later when you're looking at your parameters and trying to figure out why they've all become NaNs...)

Reply:
Yes, let's return an error instead if step has not been run.